src/condor_starter.V6.1/starter.cpp

```diff
@@ -3886,9 +3892,10 @@ Starter::removeTempExecuteDir( void )
 
 #ifdef LINUX
 	if (m_lv_handle) {
-		//LVM managed... reset handle pointer to call destructor for cleanup
-		//We can't determine if the cleanup failed or not, and need to rm the working dir
-		m_lv_handle.reset(nullptr);
+		CondorError err;
+		if ( ! m_lv_handle->CleanupLV(err)) {
+			dprintf(D_ERROR, "Failed to cleanup LV: %s\n", err.getFullText().c_str());
+		}
```

Comment:
still need to .reset the lv_handle somewhere.   Also hook this up to the process exit code.